README.md

```diff
@@ -97,6 +98,7 @@ Examples:
     goose clickhouse "tcp://127.0.0.1:9000" status
     goose vertica "vertica://user:password@localhost:5433/dbname?connection_load_balance=1" status
     goose ydb "grpcs://localhost:2135/local?go_query_mode=scripting&go_fake_tx=scripting&go_query_bind=declare,numeric" status
+    goose starrocks "user:password@/dbname?parseTime=true&interpolateParams=true" status
```

Comment:
Would this dbstring be parseable with this package, i.e., is there a schema or alias that would work?

https://github.com/xo/dburl?tab=readme-ov-file#database-schemes-aliases-and-drivers

Reply:
I think no. There are no starrocks in this package.

Reply:
So is this a blocker because of starrocks support not provided explicitly in https://github.com/xo/dburl?tab=readme-ov-file#database-schemes-aliases-and-drivers?

Reply:
Not necessarily, but it's an outlier. One day, I'd like to get rid of users having to explicitly pass a "dialect" and instead lean on the DSN string.

The dialect (and thus the driver) is an implement detail of the goose CLI that's leaking to end-users. I.e,. you shouldn't have to care whether we're using `lib/pq` or `pgx/v5` or `pgx/v4`. The only thing an end-user should care is the database technology like `postgres` which we should parse out of the connection string.

If you're using goose as a library, it's also possible to implement a `Store` and pass that to goose, which unblocks cases like this.

Lastly, I need to audit the library we're pulling in which takes time and although it's easy to add dialects, it's also a maintenance cost to maintainers.